typewise_alert.test.py

```diff
@@ -1,11 +1,68 @@
-import unittest
-import typewise_alert
-
-
-class TypewiseTest(unittest.TestCase):
-  def test_infers_breach_as_per_limits(self):
-    self.assertTrue(typewise_alert.infer_breach(20, 50, 100) == 'TOO_LOW')
-
-
-if __name__ == '__main__':
-  unittest.main()
+import unittest
+import typewise_alert
+
+class TypewiseTest(unittest.TestCase):
+  def test_infers_breach_as_per_limits(self):
+    self.assertTrue(typewise_alert.infer_breach(0, 0, 35) == 'NORMAL')
+    self.assertTrue(typewise_alert.infer_breach(35, 0, 35) == 'NORMAL')
+    self.assertTrue(typewise_alert.infer_breach(1, 0, 35) == 'NORMAL')
+    self.assertTrue(typewise_alert.infer_breach(34, 0, 35) == 'NORMAL')
+    self.assertTrue(typewise_alert.infer_breach(36, 0, 35) == 'TOO_HIGH')
+    self.assertTrue(typewise_alert.infer_breach(-1, 0, 35) == 'TOO_LOW')
+
+
+  def test_get_temperature_breachlimits(self):
+    #test for breachlimits
+    self.assertTrue(typewise_alert.limits.get_temperature_breachlimits('PASSIVE_COOLING')==(0,35))
+    self.assertTrue(typewise_alert.limits.get_temperature_breachlimits('MED_ACTIVE_COOLING')==(0,40))
+    self.assertTrue(typewise_alert.limits.get_temperature_breachlimits('HI_ACTIVE_COOLING')==(0,45))
+    self.assertFalse(typewise_alert.limits.get_temperature_breachlimits('PASSIVE_COOLING')[0]==-1)
+    self.assertTrue(typewise_alert.limits.get_temperature_breachlimits('MED_ACTIVE_COOLING')[0]==0)

```

Comment:
this condition is already checked in line 17. this seems redundant.

Reply:
Line 20 was added to test the low limit specifically.
But as you pointed out, it is already covered in line 17 with low and high limit together. I will remove this